scripts/zones/Wajaom_Woodlands/mobs/Zoraal_Ja_s_Pkuucha.lua

```diff
@@ -11,8 +13,33 @@ function onMobSpawn(mob)
 end;
 
 -----------------------------------
+-- OnMobFight Action
+-----------------------------------
+
+function onMobFight(mob, target)
+    if(mob:getHPP() <= math.random(20,50) and GetMobAction(Percipient_Zoraal_Ja) == ACTION_NONE
```

Comment:
Yeah. If you leave it in there, every second, it will roll for random 20-50, which would make it spawn the master closer to 50% way more often than at lower percent.